app/migrations/init_db.py

```diff
@@ -272,8 +272,8 @@ def init_db():
                     )
                     ''')
     cur.execute('''
-                    CREATE UNIQUE INDEX IF NOT EXISTS idx_stake_ledger_wallet 
-                    ON stake_ledger (wallet_address)
+                    CREATE UNIQUE INDEX IF NOT EXISTS idx_stake_ledger_address_person_id
+                    ON stake_ledger (address,person_id)
```

Comment:
@Prannav-k one wallet can have only one entry in stake ledger right?

Reply:
yes , for a contract , one wallet can have only one entry. But there can be 2 contracts (newrl stake and og stake), so we need contract address + pid/wallet combination.